src/map/map_state.cpp

```diff
@@ -370,7 +370,7 @@ void map_state::update(sys::state& state) {
 
 	auto zoom_diff = (zoom_change * seconds_since_last_update) / (1 / zoom);
 	zoom += zoom_diff;
-	zoom_change *= std::exp(-seconds_since_last_update * 20);
+	zoom_change *= std::exp(-seconds_since_last_update * state.user_settings.zoom_speed);
```

Comment:
test for isfinite and then clamp between two reasonable values in case of a junk settings value

Reply:
clamped on load, does this do kosher?